cristin-commons/src/main/java/no/unit/nva/cognito/CognitoHelper.java

```diff
@@ -99,13 +99,14 @@ public String createUser(String feideId, String password, String accessRight) {
      * Update user accessRights attributes.
      *
      * @param feideId user identifier
+     * @param accessRights
      * @return result of operation
      */
-    public AdminUpdateUserAttributesResult updateUserAttributes(String feideId) {
+    public AdminUpdateUserAttributesResult updateUserAttributes(String feideId, String accessRights) {
```

Comment:
In general, can you fix the return values for the methods in this class and update the Javadoc

Reply:
Also line 89 has a problem